static/js/place/place_template.tsx

```diff
@@ -512,6 +512,7 @@ interface ChartStateType {
   dataPoints?: DataPoint[];
   dataGroups?: DataGroup[];
   elemWidth: number;
+  dateSelected?: string;
```

Comment:
since titleString doesn't seem to be used, let's remove it from the class

Reply:
I don't see titleString, do you mean titleSuffix?

Reply:
oh yes, titlesuffix